SVGLayer.h

```diff
@@ -8,12 +8,76 @@
 
 #import <QuartzCore/QuartzCore.h>
 
-/// An SVG-backed CALayer that you can view that parses, renders, and display an SVG file.
+/// A CALayer subclass that renders an SVG file.
 
 @interface SVGLayer : CALayer
+
+NS_ASSUME_NONNULL_BEGIN
+
+/*!
+ *  @discussion Renders the provided SVG.
+ *
+ *  @param aSVGName The name of an SVG file in the app bundle.
+ *
+ *  @code // iOS:
+ let myLayer = SVGLayer()
+ myLayer.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+ myLayer.renderSVGNamed("tiger")
+ view.layer.addSublayer(myLayer)
+
+ // macOS:
+ let myLayer = SVGLayer()
+ myLayer.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+ myLayer.geometryFlipped = true
+ myLayer.renderSVGNamed("tiger")
+ view.layer?.addSublayer(myLayer)
+ *
+ */
+- (void)renderSVGNamed:(NSString *)aName;
```

Comment:
the only point of the loadSVGNamed: method is to allow live update of files as they're edited, while in debug mode, but that can be moved down to SVGImageView.

I think the best way is just to set an array of SVGPath* objects on the layer. (that class didn't exist when I wrote SVGLayer)

SVGView can have those more user friendly init methods (A'la UIImageView). 

initWithContentsOfFile is redundant when you have initWithContentsOfURL though :)